modules/custom-status/custom-status.php

```diff
@@ -1243,515 +1309,544 @@ function print_configure_view() {
 				<div class="col-wrap">
 				<div class="form-wrap">
 				<h3 class="nav-tab-wrapper">
-					<a href="<?php echo esc_url( $this->get_link() ); ?>" class="nav-tab<?php if ( !isset( $_GET['action'] ) || $_GET['action'] != 'change-options' ) echo ' nav-tab-active'; ?>"><?php _e( 'Add New', 'edit-flow' ); ?></a>
-					<a href="<?php echo esc_url( $this->get_link( array( 'action' => 'change-options' ) ) ); ?>" class="nav-tab<?php if ( isset( $_GET['action'] ) && $_GET['action'] == 'change-options' ) echo ' nav-tab-active'; ?>"><?php _e( 'Options', 'edit-flow' ); ?></a>
+					<a href="<?php echo esc_url( $this->get_link() ); ?>" class="nav-tab
+										<?php
+										if ( ! isset( $_GET['action'] ) || 'change-options' != $_GET['action'] ) {
+											echo ' nav-tab-active';}
+										?>
+					"><?php _e( 'Add New', 'edit-flow' ); ?></a>
+					<a href="<?php echo esc_url( $this->get_link( array( 'action' => 'change-options' ) ) ); ?>" class="nav-tab
+										<?php
+										if ( isset( $_GET['action'] ) && 'change-options' == $_GET['action'] ) {
+											echo ' nav-tab-active';}
+										?>
+					"><?php _e( 'Options', 'edit-flow' ); ?></a>
 				</h3>
-				<?php if ( isset( $_GET['action'] ) && $_GET['action'] == 'change-options' ): ?>
+				<?php if ( isset( $_GET['action'] ) && 'change-options' == $_GET['action'] ) : ?>
 				<form class="basic-settings" action="<?php echo esc_url( $this->get_link( array( 'action' => 'change-options' ) ) ); ?>" method="post">
 					<?php settings_fields( $this->module->options_group_name ); ?>
 					<?php do_settings_sections( $this->module->options_group_name ); ?>
 					<?php echo '<input id="edit_flow_module_name" name="edit_flow_module_name" type="hidden" value="' . esc_attr( $this->module->name ) . '" />'; ?>
 					<?php submit_button(); ?>
 				</form>
-				<?php else: ?>
-				<?php /** Custom form for adding a new Custom Status term **/ ?>
+				<?php else : ?>
+					<?php /** Custom form for adding a new Custom Status term **/ ?>
 					<form class="add:the-list:" action="<?php echo esc_url( $this->get_link() ); ?>" method="post" id="addstatus" name="addstatus">
 					<div class="form-field form-required">
 						<label for="status_name"><?php _e( 'Name', 'edit-flow' ); ?></label>
-						<input type="text" aria-required="true" size="20" maxlength="20" id="status_name" name="status_name" value="<?php if ( !empty( $_POST['status_name'] ) ) echo esc_attr( $_POST['status_name'] ) ?>" />
+						<input type="text" aria-required="true" size="20" maxlength="20" id="status_name" name="status_name" value="<?php /* phpcs:ignore Generic.ControlStructures.InlineControlStructure.NotAllowed */ if ( ! empty( $_POST['status_name'] ) ) echo esc_attr( $_POST['status_name'] ); ?>" />
```

Comment:
A ternary here might be easier:

```suggestion
						<input type="text" aria-required="true" size="20" maxlength="20" id="status_name" name="status_name" value="<?php echo ( empty( $_POST['status_name'] ) ? '' : esc_attr( $_POST['status_name'] ) ); ?>" />
```

Reply:
Same for the `phpcs:ignore` below

Reply:
Done